docs/App.js

```diff
@@ -78,7 +78,7 @@ class App extends Component {
           <Colorscale
             colorscale={this.state.colorscale}
             onClick={() => {}}
-            width={150}
+            width={200}
```

Comment:
if you'd kept it at 150 you would have had a preview of what it'd look like in the REC :)